specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/ConversationalAnalysis-authoring.json

```diff
@@ -0,0 +1,2742 @@
+          },
+          "401": {
+            "description": "You do not have access. \\r\\n\\r\\nReasons can include:\\r\\n\\r\\n* used endpoint subscription key, instead of authoring key\\r\\n* invalid, malformed, or empty authoring key\\r\\n* authoring key doesn't match region\\r\\n* you are not the owner or collaborator\\r\\n* invalid order of API calls."
+          },
+          "403": {
+            "description": "Total monthly key quota limit exceeded."
+          },
+          "429": {
+            "description": "Rate limit is exceeded."
+          }
+        }
+      }
+    },
+    "/projects/{projectName}": {
+      "patch": {
+        "description": "Creates a new or update a project.",
+        "operationId": "Projects_CreateOrUpdateProject",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "projectName",
+            "description": "The project name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "projectCreateObject",
+            "description": "The project parameters.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ProjectCreateObject"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The metadata of the updated project, if it already exists.",
+            "schema": {
+              "$ref": "#/definitions/ProjectGetObject"
+            }
+          },
+          "201": {
+            "description": "The metadata of the created project.",
+            "schema": {
+              "$ref": "#/definitions/ProjectGetObject"
+            }
+          },
+          "400": {
+            "description": "This error can be returned if the request's parameters are incorrect meaning the required parameters are missing, malformed, or too large.\\r\\n\\r\\nThis error can be returned if the request's body is incorrect meaning the JSON is missing, malformed, or too large."
+          },
+          "401": {
+            "description": "You do not have access. \\r\\n\\r\\nReasons can include:\\r\\n\\r\\n* used endpoint subscription key, instead of authoring key\\r\\n* invalid, malformed, or empty authoring key\\r\\n* authoring key doesn't match region\\r\\n* you are not the owner or collaborator\\r\\n* invalid order of API calls."
+          },
+          "403": {
+            "description": "Total monthly key quota limit exceeded."
+          },
+          "429": {
+            "description": "Rate limit is exceeded."
+          }
+        }
+      },
+      "get": {
+        "description": "Gets a project info.",
+        "operationId": "Projects_GetProject",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
```

Comment:
None of the API have `api-version` query parameter?

I see `api-version` property in payload of PUT/POST. However how do you handle GET request?